packages/xstate-vue/test/UseInterpret.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <button @click="service?.send('TOGGLE')" data-testid="button">
+    {{ state === 'inactive' ? 'Turn on' : 'Turn off' }}
+  </button>
+</template>
+
+<script lang="ts">
+import { useInterpret } from '../src';
+import { createMachine } from 'xstate';
+
+import { defineComponent, ref } from 'vue';
+const machine = createMachine({
+  initial: 'inactive',
+  states: {
+    inactive: {
+      on: {
+        TOGGLE: 'active'
+      }
+    },
+    active: {
+      on: {
+        TOGGLE: 'inactive'
+      }
+    }
+  }
+});
+
+export default defineComponent({
+  setup() {
+    const service = useInterpret(machine);
+    const state = ref();
+    service.subscribe((nextState) => {
+      state.value = nextState.value;
+    });
+    return { service, state };
+  }
```

Comment:
To kinda utilize the `useInterpret`'s API better you could this:
```suggestion
    const state = ref();
    const service = useInterpret(machine, {}, (nextState) => {
      state.value = nextState.value;
    });
    return { service, state };
  }
```